cli/register/register.go

```diff
@@ -193,13 +193,15 @@ func DoIt(org, pattern, nodeIdTok, userPw, inputFile string, nodeOrgFromFlag str
 	exchangePattern := ""
 	httpCode := cliutils.ExchangeGet("Exchange", exchUrlBase, "orgs/"+org+"/nodes/"+nodeId, cliutils.OrgAndCreds(org, nodeIdTok), nil, &devicesResp)
 
+	cliutils.SetWhetherUsingApiKey(userPw)
+	userOrg, userAuth := cliutils.TrimOrg(org, userPw)
```

Comment:
User id and password may not be specified in `hzn register` command.  There are two ways to specify the credentials for this command. Either -u userid:userpasswd or -n nodeid:token, or both.  If the node does not exist, -u must be specified in order to create the node. If the node exists, but do not know the node token, -u must be specified  in order to change the token.  If node exists and you know the token, you can just specify -n for this command.